customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDao.java

```diff
@@ -126,13 +129,21 @@ public void setInactiveFrom(Instant inactiveFrom) {
         this.inactiveFrom = inactiveFrom;
     }
 
+    public boolean isAllowsGeneralSupport() {
```

Comment:
There may be better name for this variable so getter is more readable.

Reply:
`allowsGeneralSupport` is probably what we are looking for — does this conflict? Maybe

Reply:
or `generalSupportEnabled`, which makes the setter look like this: `isGeneralSupportEnabled()`

Reply:
I agree, but pmd does not like that getter for boolean variable is named without `is` in method name.